### PR TITLE
Add biom-format 2.1.7

### PR DESCRIPTION
**Pull request by wasade**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/biom-format/2.1.7/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+source:
+  url: https://github.com/biocore/biom-format/archive/2.1.7.tar.gz
+  md5: c4d2a0ebfe98bf174e482e97937e1ef6
+
+
+build:
+  preserve_egg_dir: True
+  entry_points:
+    - biom=biom.cli:cli
+  number: 1
```
Comment (bgruening) [inline recipes/biom-format/2.1.7/meta.yaml]: please start with 0

**Comment by bgruening** [comment]
Merci!

**Comment by wasade** [comment]
Thank you!

**Comment by wasade** [comment]
@bgruening, we just discovered that only py27 is available, any idea why py3 builds aren't coming through?

cc @thermokarst

**Comment by druvus** [comment]
It is only building py27 because the recipe contains this line`skip: true  # [py3k]`. Remove it in a new PR to also build for py3.

**Comment by wasade** [comment]
Okay, thanks. Do you know why the other builds of biom, which have this,
also have py3 builds?

**Pull request merged**